experimental/Schemes/elliptic_surface.jl

```diff
@@ -2103,13 +2106,27 @@ end
 # The map is not dominant and can hence not be realized as a MorphismFromRationalFunctions.
 # We keep the code for the moment as it will probably help us to reconstruct this map as a 
 # proper CoveredSchemeMorphism, once this is needed. 
-function morphism_from_section(X::EllipticSurface, i::Int)
-  P = X.MWL[i]
-  @assert !P.is_infinite
+=#
+function morphism_from_section(
+    X::EllipticSurface, P::EllipticCurvePoint;
+    divisor::AbsWeilDivisor=_section(X, P)
+  )
```

Comment:
This works now, but returns two maps: The identification IP^1 with the section and the inclusion of the latter.